apps/fares/lib/format.ex

```diff
@@ -87,8 +87,11 @@ defmodule Fares.Format do
   def name({:zone, zone}), do: "Zone #{zone}"
   def name({:interzone, zone}), do: "Interzone #{zone}"
   def name(:foxboro), do: "Foxboro Special Event"
-  # A free fare might be an SL1 trip from airport stops or shuttle bus service
+  # A free fare might be a shuttle bus service
   def name(:free_fare), do: "Free Fare"
```

Comment:
Quick point before looking at the rest: I believe we want to change the wording to "Free Service" in all cases, not just airport trips.

Reply:
Right now it says "Shuttle: $0.00" when I look up a shuttle trip, will that stay the same or change also to Free Service? I like the specificity of "Shuttle: $0.00".

Reply:
@arkadyan, I changed the logic a bit so the string is the only thing that changes from `"Free Fare"` to `"Free Service"`.
@ingridpierre, which A and B points were you entering? there should be a distinction between `"Shuttle"` and SL1's `"Free Service"`.

Reply:
@alicia-mbta I think we're good then! I didn't see any shuttle fares change. I think I was just confused as to what this change would affect.